Civi/Core/Container.php

```diff
@@ -389,6 +389,10 @@ public function createContainer() {
       'CRM_Core_DomainTokens',
       []
     ))->addTag('kernel.event_subscriber')->setPublic(TRUE);
+    $container->setDefinition('crm_site_tokens', new Definition(
```

Comment:
It's not quite merged yet but when this is merged https://github.com/civicrm/civicrm-core/pull/30461 it won't be necessary to do this as it can be declared in the entityType